bin/load_test

```diff
@@ -40,7 +40,7 @@ then
   mkvirtualenv locust && \
     workon locust &&  \
     pip install -r scripts/load_testing/requirements.txt && \
-    locust -f scripts/load_testing/$test_to_run.py -H http://localhost:3000 --no-web -c 3 -n 27 -r 1
+    locust -f scripts/load_testing/$test_to_run.py -H https://idp.pt.login.gov --no-web -c 10 -n 90 -r 10
```

Comment:
I don't think this change should go into master. The host should be set to localhost by default and can be modified manually on the user's machine if they need to target a different host.